metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -165,13 +166,13 @@
 (defn- provider-ids-for-validation
   "Returns the set of provider-ids for validation purpose. It is a list of existing provider ids
   and 'CMR', which is reserved for tags / tag associations."
-  [db]
-  (set (conj (map :provider-id (provider-db/get-providers db)) "CMR")))
+  [providers]
```

Comment:
now that we no longer take a context or db object, we can unit test these functions and get a code coverage bump

Reply:
a microscopic bump :-).  This code is being tested by both the DB integration tests as well as the system integration tests.  

Is there a way we can include these tests in the code coverage count?

I will add a couple of tests.